react-native/react/local-debug.desktop.js

```diff
@@ -18,8 +18,8 @@ let config = {
   reduxDevToolsEnable: false,
   redirectOnLogout: true,
   reduxDevToolsSelect: state => state, // only watch a subset of the store
-  enableStoreLogging: false,
-  forwardLogs: false
+  enableStoreLogging: true,
+  forwardLogs: true
```

Comment:
i say we turn it off for now, ppl can turn it on if we need help debugging. longer term we could use something like winston: https://github.com/winstonjs/winston